projects/demo/src/modules/info/testing/disable-animation/disable-animation.template.html

```diff
@@ -15,32 +12,65 @@
     </p>
 
     <p>
-        For example, we use
-        <a
-            href="https://www.cypress.io/"
-            rel="noreferrer"
-            target="_blank"
-            tuiLink
-        >
-            Cypress
-        </a>
-        for screenshot testing of our ui-kit components.
+        <code>&#64;taiga-ui/cdk</code>
+        used
+        <ng-container *ngIf="'TUI_REDUCED_MOTION' | tuiDocTypeReference as refs">
+            <code *ngFor="let ref of refs">
```

Comment:
*ngFor="let ref of 'TUI_REDUCED_MOTION' | tuiDocTypeReference"

Reply:
fixed